podcastfy/client.py

```diff
@@ -71,14 +71,20 @@ def process_content(
                 qa_content = file.read()
         else:
             content_generator = ContentGenerator(
-                api_key=config.GEMINI_API_KEY, conversation_config=conv_config.to_dict()
+                api_key=config.OPENAI_API_KEY, conversation_config=conv_config.to_dict()
             )
 
             if urls:
                 logger.info(f"Processing {len(urls)} links")
                 content_extractor = ContentExtractor()
-                # Extract content from links
-                contents = [content_extractor.extract_content(link) for link in urls]
+                # Extract content from links or file paths
```

Comment:
it's better to add the below logic (test whether it's a link or file path) to inside extract_content and keep this as list comprehension